frontend/src/actions/user_actions.js

```diff
@@ -0,0 +1,28 @@
+import * as UserAPI from '../util/user_api_util';
+
+export const RECEIVE_USERS = 'RECEIVE_USERS';
+export const RECEIVE_USER_ERRORS = 'RECEIVE_USER_ERRORS';
+export const CLEAR_USER_ERRORS = 'CLEAR_USER_ERRORS';
+
+export const clearUserErrors = () => ({
+  type: CLEAR_USER_ERRORS,
+});
+
+export const receiveUserErrors = (errors) => ({
+  type: RECEIVE_USER_ERRORS,
+  errors,
+});
+
+export const receiveUsers = (users) => ({
+  type: RECEIVE_USERS,
+  users,
+});
+
+export const fetchUsers = () => (dispatch) => {
+  UserAPI.fetchUsers()
```

Comment:
Do we want to return the fetch promise here? I've occasionally found it useful to have in components

Reply:
Good call- just a `return` right?

Reply:
yeah